BoxContentSDK/BoxContentSDK/Requests/BOXCollaborationCreateRequest.m

```diff
@@ -8,12 +8,19 @@
 #import "BOXAPIJSONOperation.h"
 #import "BOXCollaboration.h"
 
+@interface BOXCollaborationCreateRequest()
+
+@property (nonatomic, readwrite, strong) BOXAPIItemType *itemType;
```

Comment:
or maybe I'm wrong, I thought you could not use ivar without readwrite, but I guess I was mistaken ?